src/site/docs/dart-up-and-running/ch05.markdown

```diff
@@ -433,40 +432,39 @@ void handleWebSocket(WebSocket webSocket) {
 
 The call to `webSocket.map()` parses all the messages that the client
 sends over the WebSocket, converting each JSON-formatted message into an
-object. Then, after checking the message format, the handler initiates
-searches on GitHub and StackOverflow.
+object. Then, after checking the message format, the handler calls
+the helper function `_getResults()` which initiates
+searches on GitHub and StackOverflow, constructs a JSON-formatted
+message for each result, and uses `webSocket.addStream()` to forward
+the result to the client.
 
-Here's the code from the 'search' case that starts the searches and
-handles results as they come in:
+Here's the code from the `_getResults()` case that starts the searches and
+yields results as they come in:
```

Comment:
yields -> handles

The functionality hasn't changed, just the implementation, right? it seems like we should use the same words to describe the functionality, although we should also talk (probably after the code) about *how* it does it.